launch/camera2.launch

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<launch>
+
+    <!-- Produced topics -->
+    <arg name="object_markers"  default="object_markers"/>
+
+    <!-- Options -->
+    <arg name="use_cuda"            default="true"/>
+    <arg name="enable_bbox_image"   default="false"/>
+    <arg name="enable_aruco"        default="false"/>
+    <arg name="is_sim"              default="true"/>
+    <arg name="map"         default="floor2"/>
+
+    <!-- Auxiliary -->
+    <arg name="max_age"             default="30"/>
+    <arg name="aruco_size"          default="0.360"/>
+    <rosparam command="load" file="$(find svea_core)/params/obstacles.yaml" />
+
+    <!-- Constants -->
+    <arg name="depth_image"     value="/zed/zed_node/depth/depth_registered"/>
+    <arg name="image"           value="/zed/zed_node/rgb/image_rect_color"/>
+    <arg name="websocket_port"  value="9090"/>
+
+
+    <arg name="map_img_name" default="$(find svea_core)/maps/floor2.yaml"/>
+    <node pkg="map_server" name="map_server" type="map_server" args="$(arg map_img_name)"/>
+
+    <!-- Nodes -->
+    <include file ="$(find el2425_standards)/launch/infra1.launch"/>
+    <include file="$(find zed_wrapper)/launch/zed.launch">
+        <arg name="camera_name" value="zed"/>
+    </include>
+
+    <include file="$(find svea_vision)/launch/object_pose.launch">
+        <!-- Options -->
+        <arg name="use_cuda"            value="$(arg use_cuda)"/>
+        <arg name="enable_bbox_image"   value="$(arg enable_bbox_image)"/>
+        <arg name="frame_id"            value="$(arg frame_id)"/>
+        <!-- Consumed topics -->
+        <arg name="image"               value="$(arg image)"/>
+        <arg name="depth_image"         value="$(arg depth_image)"/>
+        <!-- Auxiliary -->
+        <arg name="only_objects"        value="$(arg only_objects)"/>
+        <arg name="skip_objects"        value="$(arg skip_objects)"/>
+        <arg name="max_age"             value="$(arg max_age)"/>
+    </include>
+
+    <include if="$(arg enable_aruco)" file="$(find svea_vision)/launch/aruco_detect.launch">
+        <!-- Consumed topics -->
+        <arg name="image"       value="$(arg image)"/>
+        <!-- Auxiliary -->
+        <arg name="aruco_size"  value="$(arg aruco_size)"/>
+    </include>
+
+    <arg name="node_start_delay" default="30.0" />
+
+    <!-- FOR CALIBRATION: 
+    1. COMMENT "camera2map" NODE AND UNCOMMENT "aruco2map" NODE
+    2. READ POSITION OF ARUCO IN WORLD COORDS AND ENTER THEM INTO args = "x y z quatx quaty qautz quatw map aruco[index]" -->
+    <node name="camera2map" pkg="infra" type="tf_camera2map.py" output="screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' " ></node>
```

Comment:
1. Do we need this node?
2. If we do, then add the script to this PR

Reply:
Automatic calibration is nice, but is it this node that does it?

Reply:
I haven't included those, I was planning to do that today.

My question is if we need **aruco2map**, it seems to be designed specifically for handling the poses of aruco markers, while **Camera2map** is for a broader set of object poses.

**Camera2map**: is used to transform from the zed camera coordinate frame to the global map coordinate frame.

**Camera2**: is used to transform from second camera to map frame. (there are two cameras recording, the zed camera and one more)

**merge_cameras.py**:  is used to combine measurements from both cameras after they have been
transformed from the camera coordinate frame to the global map coordinate frame.

Reply:
No no, as far as I understand these nodes do not have automatic calibration, they are just for transforming the camera coordinates to global map coordinates.

I think they manually performed the calibration, but maybe I am wrong